chpl/chpl-service/src/main/java/gov/healthit/chpl/util/CertificationResultRules.java

```diff
@@ -165,6 +165,13 @@ private void setRulesUsingDatabase(CertificationCriterionAttributeDAO certificat
                         .optionName(TEST_TOOLS_USED)
                         .build());
             }
+            if (attribute.getTestData()) {
+                rules.get(attribute.getCriterion().getNumber()).add(
+                        CertificationResultOption.builder()
+                        .canHaveOption(true)
```

Comment:
Do we ever call this builder with `.canHaveOption(false)`, or even `.canHaveOption(aVariable)`? If not, would it make sense / reduce code to just default this to `true`? Is that even possible? Or worth it?